src/api/queues.js

```diff
@@ -120,18 +120,36 @@ router.patch(
   safeAsync(async (req, res, _next) => {
     const { queue } = res.locals
     const data = matchedData(req)
-
     await queue.update({
       name: data.name,
       location: data.location,
     })
     const updatedQueue = await Queue.scope('questionCount').findOne({
       where: { id: queue.id },
     })
+
     res.status(201).send(updatedQueue)
   })
 )
 
+router.patch(
```

Comment:
If we want to stick to proper HTTP semantics (we already violate them in some places, but) we should really just have a single `PATCH` endpoint that can handle updating name, location, open/closed status, etc; it just updates any fields that are present in the request.